nrfe/app/nrfe_gpio.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFE_GPIO_H
+#define NRFE_GPIO_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief FLPR EGPIO opcodes. */
+typedef enum
+{
+    NRFE_GPIO_PIN_CONFIGURE	= 0, ///< Configure eGPIO pin.
+    NRFE_GPIO_PIN_CLEAR    	= 1, ///< Clear eGPIO pin.
+    NRFE_GPIO_PIN_SET      	= 2, ///< Set eGPIO pin.
+    NRFE_GPIO_PIN_TOGGLE   	= 3, ///< Toggle eGPIO pin.
+} egpio_opcode_t;
+
+/** @brief EGPIO data packet. */
+struct nrfe_gpio_data_packet {
```

Comment:
Let's stick to nrfx coding standard for now, so this would be an anonymous struct typedefed into `nrfe_gpio_data_packet_t`.